swim/node.go

```diff
@@ -510,3 +510,7 @@ func (n *Node) pingNextMember() {
 		"numErrors": len(errs),
 	}).Warn("ping request inconclusive due to errors")
 }
+
```

Comment:
Is there a reason why you didn't propagate the name 'GetActiveMemberAddresses' upwards?

I was thinking in place of using 'ActiveMember', which is introduces a new term into our Ringpop vocabulary, using 'GetPingableMembers'? What do you think?